Makefile

```diff
@@ -95,10 +95,25 @@ vagrant-sync-optional:
 vagrant-deploy: vagrant-sync-config vagrant-sync-build
 	export KUBECTL="cluster/kubectl.sh --core" && ./cluster/deploy.sh
 
-.release-functest:
+functest-release: kubevirt.yaml
+	make vagrant-deploy DOCKER_TAG=$(TAG) DEPLOY_MANIFEST=$$PWD/kubevirt.yaml
 	make functest > .release-functest 2>&1
 
-release-announce: .release-functest
+kubevirt.yaml: TAG=$(shell git describe)
+kubevirt.yaml: PREFIX=kubevirt
+kubevirt.yaml:
+	echo "## added-for-release" >> hack/config-local.sh
```

Comment:
config-local overrides the default config, which is the functionality you're wanting.

However, config-local isn't cleaned up, which means this could cause some unexpected and difficult to detect behavior in someone's dev environment after they run 'make kubevirt.yaml'

also, the first echo to the config-local file probably needs > instead of >> in order to truncate any data that might have already been present.

Reply:
See below, that is what sed is doing.